styles/modules/modals/_editListing.scss

```diff
@@ -39,6 +39,10 @@
 
     .tabContent {
       .photoUploadSection {
+        h2 {
+          display: inline-block;
```

Comment:
You could just put a floL class on the h2, and then you wouldn't need this rule.